README.md

```diff
@@ -100,10 +100,27 @@ Config options overview:
 | password | smtp    |                             | Password for email client.                                    |
 | host     | smtp    | smtp.gmail.com              | Host of the email client.                                     |
 
+### Transformation configuration
+
+Configuration files for TranSMART and cBioPortal must be placed in `transformation_config_dir`. 
+Specifically, the following are expected:
+
+- `sources_config.json` and `ontology_config.json`, described in [python_csr2transmart](https://github.com/thehyve/python_csr2transmart#usage); 
+the files reference the input data and need to be customized accordingly,
+- `portal.properties` file for cBioPortal; 
+the file  must match the mounted cBioPortal image version and server environment, 
+- `cbioportal_db_info` folder, containing configuration files for the cBioPortal database 
+(`cancertypes.json`, `genes.json`, `genesaliases.json`); 
+the files must match the mounted cBioPortal image version.
+ 
+Sample configuration files are provided in [test_data/test_data_NGS/config](https://github.com/thehyve/pmc-conversion/test_data/test_data_NGS/config).
```

Comment:
@ewelinagr do you think the link will work like this? in your example I saw tree/master/ was also part of the path, but I guess it would default to master anyway?

Reply:
unfortunately not, it will give '404 - page not found'